docs/program/apis/sessions.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Session Management with Viam's Client SDKs"
+linkTitle: "Session Management API"
+weight: 20
+type: "docs"
+description: "How to use the session management API with Viam's Client SDKs."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session", "sessions", "session management"]
+---
+
+The session management API provides support for robot session management.
+
+A session:
+
+- allows a client to express that it is actively connected or authenticated to `viam-server` on your robot
+- supports stopping moving components when the connection is no longer active.
+
+### Purpose of session management
+
+When controlling a robot or fleet with Viam, you want a way to understand the presence of the clients that are communicating with and authenticated to each robot's `viam-server` through Viam's various [resource APIs](/program/apis/).
+
+The presence of these clients at one point in time is called a *session*.
```

Comment:
Per my comment on the last go 'round, is this right? I think it's actually a period of time, not the presence of something at a point in time, right?

Reply:
Great point-- this was weird phrasing I copied-- LMK if you think updated phrasing is more sensible